plugins/auth-node/api-report.md

```diff
@@ -639,6 +639,20 @@ export type TokenParams = {
   } & Record<string, JsonValue>;
 };
 
+// @public
+export const tokenTypes: Readonly<{
```

Comment:
i was going back and forth on this one - whether it was worth going for namespaces instead, and making it possible to have for example `TokenTypes.isUserToken(t: string)` and `TokenTypes.User.Payload` and stuff like that

Reply:
Sure!